ci_framework/roles/cifmw_cephadm/tasks/cephfs.yml

```diff
@@ -0,0 +1,48 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Build placement string
+  block:
+    - name: Collect the host and build the resulting host list
+      ansible.builtin.set_fact:
+        _hosts: "{{ _hosts|default([]) + [ hostvars[item]['ansible_hostname'] ] }}"
+      with_items: "{{ groups['edpm'] }}"
+
+    - name: Collect the edpm hosts
+      ansible.builtin.set_fact:
+        placement: "{{ ''.join((placement|default(''), (item+' '))) }}"
+      with_items: "{{ _hosts | unique }}"
+
+- name: Get ceph_cli
+  ansible.builtin.include_tasks: ceph_cli.yml
+
+- name: Apply cephfs volume
+  ansible.builtin.command: |
+    {{ cifmw_cephadm_ceph_cli }} fs volume create {{ cifmw_cephadm_cephfs_name }} '--placement={{ placement }}'
+  changed_when: false
+  become: true
+
+# waiting for https://github.com/ceph/ceph/pull/53108
+# to appear in the next Ceph container build
+# disabling this task by default for now
+- name: Apply cephfs volume
+  when: ceph_nfs | default(false)
```

Comment:
@fultonj @gouthampacha @katarimanojk we remove this condition here as long as the `ceph-nfs` bits are available in cephadm.

Reply:
@fmount Ack